lightning/src/routing/gossip.rs

```diff
@@ -1130,13 +1130,13 @@ impl<L: Deref> Writeable for NetworkGraph<L> where L::Target: Logger {
 		self.genesis_hash.write(writer)?;
 		let channels = self.channels.read().unwrap();
 		(channels.len() as u64).write(writer)?;
-		for (ref chan_id, ref chan_info) in channels.iter() {
+		for (ref chan_id, ref chan_info) in channels.unordered_iter() {
```

Comment:
Will the randomized order have any consequences for RGS server processing of the NetworkGraph?

Reply:
Specifically, look here: https://github.com/lightningdevkit/rapid-gossip-sync-server/blob/f0ef17314840b4e30d08511dfc7773b4ffc8d343/src/lookup.rs#L18

Reply:
Ah, so it's using its own ordered `BTreeMap`, makes sense. Thanks!